shared/store/wallet.ts

```diff
@@ -84,10 +86,18 @@ export const fetchPoolPairs = createAsyncThunk(
 
 export const fetchTokens = createAsyncThunk(
   'wallet/fetchTokens',
-  async ({ size = 200, address, client }: { size?: number, address: string, client: WhaleApiClient }): Promise<{ tokens: AddressToken[], utxoBalance: string }> => {
+  async ({ size = 200, address, client }: { size?: number, address: string, client: WhaleApiClient }): Promise<{ tokens: WalletToken[], utxoBalance: string }> => {
     const tokens = await client.address.listToken(address, size)
+    if (!tokens.some((t) => t.id === '0')) {
+      tokens.push(tokenDFI)
+    }
+    const detailedToken: WalletToken[] = await Promise.all(tokens.map(async (token) => {
+      const activePrices = await client.prices.getFeedActive(token.symbol, 'USD', 1)
+      const detailToken = setTokenDetails(token)
```

Comment:
I find the `setTokenDetails` method name to be vague, can we try to improve the naming to be more explicit?
If it is only changing the symbol, then why not call it `setTokenSymbol`?